grid2op/tests/test_RewardAlertCostScore.py

```diff
@@ -0,0 +1,373 @@
+# Copyright (c) 2023, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import warnings
+import numpy as np
+import unittest
+import tempfile
+import grid2op
+from grid2op.Reward import _AlertCostScore, _AlertTrustScore
+from grid2op.Agent import DoNothingAgent, BaseAgent
+from grid2op.tests.helper_path_test import *
+from grid2op.Exceptions import Grid2OpException
+from grid2op.Runner import Runner 
+from grid2op.Observation import BaseObservation
+from grid2op.Episode import EpisodeData
+from grid2op.Parameters import Parameters
+from grid2op.Opponent import BaseOpponent, GeometricOpponent
+from grid2op.Action import BaseAction, PlayableAction
+from _aux_opponent_for_test_alerts import (_get_steps_attack,
+                                           TestOpponent
+                                           )
+
+ATTACKED_LINE = "48_50_136"
+
+class AlertAgent(BaseAgent):
+    def act(self, observation: BaseObservation, reward: float, done: bool = False) -> BaseAction:
+        if observation.current_step == 2:
+            return self.action_space({"raise_alert": [0]})
+        return super().act(observation, reward, done)
+
+
+class TestAlertCostScore(unittest.TestCase):
+    
+    def test_specs(self):
+        # test function without actual data
+        assert _AlertCostScore._penalization_fun(50) == -1.
+        assert _AlertCostScore._penalization_fun(80) == 0.
+        assert _AlertCostScore._penalization_fun(100) == 1.
+    
+    def setUp(self) -> None:
+        self.env_nm = os.path.join(
+            PATH_DATA_TEST, "l2rpn_idf_2023_with_alert"
+        )
+        
+    def tearDown(self) -> None:
+        return super().tearDown()
+        
+    def test_assistant_reward_value_no_blackout_no_attack_no_alert(self) -> None : 
```

Comment:
Only one test for this particular (and if I understood correctly really important) function ? this is ... very little

Reply:
see test_alert_trust_score.py